packages/salesforcedx-vscode-lwc-next/src/constants.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+export const ESLINT_NODEPATH_CONFIG = 'eslint.nodePath';
+export const LWC_EXTENSION_NAME = 'salesforcedx-vscode-lwc';
```

Comment:
should  LWC_EXTENSION_NAME  be 'salesforcedx-vscode-lwc-next'?

Reply:
Yes. Good catch!